README.md

```diff
@@ -1 +1,49 @@
-# Hello-microverse-project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> A simple project to demonstrate the GitHub flow at Microverse.
+
+**Hello Microverse** is a web page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> HAFED EFEHIJ
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/hafedEfheij)
+)
```

Comment:
You can do the following things to customize this section:
1. Remove this:
❌  HAFED EFEHIJ

2. Change this  `👤 **Author**`
to:
`👤 **HAFED EFEHIJ**`


3. Change this:  `GitHub: [@githubhandle](https://github.com/hafedEfheij)`
to this:
` GitHub: [@hafedEfheij](https://github.com/hafedEfheij)`